extensions/guacamole-vault/modules/guacamole-vault-hv/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
+    "CONNECTION_ATTRIBUTES" : {
+        "SECTION_HEADER_HV_CONFIG"            : "Hashicorp Vault",
+        "FIELD_HEADER_HV_USER_CONFIG_ENABLED" : "Allow user-provided HV configuration"
+    },
+
+    "USER_ATTRIBUTES" : {
+        "SECTION_HEADER_HV_CONFIG" : "Hashicorp Vault",
+        "FIELD_HEADER_HV_CONFIG"   : "HV Service Configuration "
+    },
+
+    "CONNECTION_GROUP_ATTRIBUTES" : {
+        "SECTION_HEADER_HV_CONFIG" : "Hashicorp Vault",
+        "FIELD_HEADER_HV_CONFIG"   : "HV Service Configuration ",
+
+        "ERROR_INVALID_HV_CONFIG_BLOB" : "The provided base64-encoded HV configuration blob is not valid."
+    }
+
+}
```

Comment:
Items in this file should be in alphabetical order - so:
* `CONNECTION_GROUP_ATTRIBUTES` should be above `USER_ATTRIBUTES`
* `ERROR_` entries should be above `FIELD_` entries, which should be above `SECTION_` entries.